packages/components/Notification/src/Notification.tsx

```diff
@@ -46,9 +46,9 @@ export const Notification = compose<NotificationType>({
       return { marginHorizontal: marginHorizontal };
     }, ['isBar']);
     const messageStyle: ViewStyle = useMemo(() => {
-      const alignSelf = isBar ? 'center' : 'flex-start';
+      const alignSelf = userProps.onActionPress ? 'flex-start' : 'center';
       return { alignSelf: alignSelf };
-    }, ['isBar']);
+    }, ['userProps']);
```

Comment:
Why this change? I'd also rather pull out `const onActionPress = userProps.onActionPress` and use that here. I think `useMemo` is more efficient when the keys in the dependency array are simpler objects.

Reply:
After talking to Jeanie, she mentioned that bar notifications can also have action buttons so the alignment of the message no longer depends on variant. Rather, it depends on whether or not there is an action (onActionPress)

Reply:
Oh ok, so it's a tiny change that is paired with your bigger change, makes sense. Note about pulling out `onActionPress` still holds too then

Reply:
Cool, thanks!